roles/contiv_network/defaults/main.yml

```diff
@@ -16,6 +16,8 @@ bgp_port: 179
 vxlan_port: 4789
 netplugin_rule_comment: "contiv network traffic"
 
+aci_gw_version: "contiv/aci-gw"
+aci_gw_image: "latest"
```

Comment:
@gaurav-dalvi you probably meant to have these default values assigned the other way?